Source/Krypton Components/Krypton.Toolkit/Controls Visuals/Notification Toast/Basic/Left to Right/VisualToastNotificationBasicForm.cs

```diff
@@ -212,6 +214,20 @@ private void VisualToastNotificationBasicForm_Load(object sender, EventArgs e)
             _soundPlayer?.Play();
         }
 
+        private void UpdateSizing()
+        {
+            // Add some height, if form border style equals 'None'
+
+            var width = Size.Width;
```

Comment:
These are only needed inside the if statement.

Reply:
Now inside the `if` statement